readme.md

```diff
@@ -752,6 +752,11 @@ Featured playlists: `Network Direction`, `Network Fundamentals`, `Firewall and L
 Content about: Networking, CCNA \
 Featured playlists: `Full Series | 200-301 CCNA | Free Cisco Video Training 2020 | Networking Inc.`, `other networking videos`.
 
+[<img align="left" height="94px" width="94px" alt="NetworKing channel's avatar" src="https://yt3.ggpht.com/a/AATXAJz94lPJiSkw20P0-6TLSo4oz0Z_BQkhMiHBoZyF2w=s100-c-k-c0xffffffff-no-rj-mo"/>](https://www.youtube.com/c/PowerCertAnimatedVideos)
+[**PowerCert Animated Videos**](https://www.youtube.com/c/PowerCertAnimatedVideos) [<img height="16px" width="16px" alt="Badge for verified YouTube channels" src="media/badge-verified.svg" title="Is a verified YouTube channel"/>](badges.md#verified-youtube-channel) \
+Content about: Various networking hardware with other concepts, like protocols, sprinkled in. All the while, helpful graphics help the visual learner go along. There is also a crash course on A+ and Net+. \
```

Comment:
The "Content about" section shouldn't be larger than one line. This field should be short and concise, just one line.